.github/CODEOWNERS

```diff
@@ -18,13 +18,10 @@ stream-chat-android-test                    @GetStream/android-developers-core
 stream-chat-android-ui-common               @GetStream/android-developers-ui
 stream-chat-android-ui-components           @GetStream/android-developers-ui
 stream-chat-android-ui-components-sample    @GetStream/android-developers-ui
```

Comment:
I've simplified this a bit as some of the modules use Compose and XML at the same time.